x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_ops_stats_collector.js

```diff
@@ -56,8 +56,8 @@ export function getOpsStatsCollector(server) {
   return collectorSet.makeStatsCollector({
     type: KIBANA_STATS_TYPE_MONITORING,
     init: start,
-    fetch: () => {
-      return buffer.flush();
+    fetch: async () => {
+      return await buffer.flush();
```

Comment:
If `buffer.flush()` returns a promise, then isn't this fundamentally the same as the previous code? A returned promise will have to be awaited by the caller either way, right?